Description: Fixes test_node_attribute (blockdiag.tests.test_builder_node.TestBuilderNode).
Author: Kouhei Maeda <mkouhei@palmtb.net>
Last-Update: 2021-01-31

Index: blockdiag-2.0.1+dfsg/src/blockdiag/tests/test_builder_node.py
===================================================================
--- blockdiag-2.0.1+dfsg.orig/src/blockdiag/tests/test_builder_node.py	2021-01-31 23:21:39.398775744 +0900
+++ blockdiag-2.0.1+dfsg/src/blockdiag/tests/test_builder_node.py	2021-01-31 23:22:15.638774258 +0900
@@ -97,8 +97,8 @@
         orientations = defaultdict(lambda: 'horizontal')
         orientations['J'] = 'vertical'
         backgrounds = defaultdict(lambda: None)
-        backgrounds['K'] = ('src/blockdiag/tests/diagrams/'
-                            'debian-logo-256color-palettealpha.png')
+        backgrounds['K'] = 'src/blockdiag/tests/diagrams/debian-logo-256color-palettealpha.png'
+                            
 
         diagram = self.build('node_attribute.diag')
         self.assertNodeLabel(diagram, labels)
